examples/todo-mvc/ui/views/create_task_view.slint

```diff
@@ -34,6 +34,7 @@ export component CreateTaskView {
         HorizontalLayout {
             IconButton {
                 icon: Icons.close;
+                accessible-label: @tr("close");
```

Comment:
Perhaps this should be a little more description in saying _what_ is closed? (like a button to play music should say "play music")

@DataTriny What do you think?

(the same comment applies to most of the other labels as well, of course)

Reply:
Accessible names should be capitalized just like any user-facing text. Of course people using speech systhesis won't notice, but others might use a Braille display for instance.
I would probably add a description saying that this button goes back to the task list.

Reply:
Actually, I think "Cancel" would also be fine. It both conveys the fact that the current action will not be performed, and that the user will be brought back to the previous screen.